enigma/jwt/claims.go

```diff
@@ -0,0 +1,103 @@
+package jwt
+
+import (
+	"encoding/json"
+	"github.com/pborman/uuid"
+	"time"
+)
+
+// Claims represent a token's claims.
+type Claims struct {
```

Comment:
@leetal I removed `NewClaimsContext()` because it's very hard to get all the parameters right as most of them are strings. I think taking the struct approach is more prone to developer errors